frontend/explorer/public/js/main.js

```diff
@@ -254,8 +348,28 @@ function appendRawBlock(element, explorerBlock) {
 }
 
 function appendExplorerBlock(element, explorerBlock) {
+	appendNavigationButtons(element, explorerBlock);
+	appendBlockSearchField(element, explorerBlock);
 	appendBlockStatistics(element, explorerBlock);
 	appendBlockMinerPayouts(element, explorerBlock);
 	appendBlockTransactions(element, explorerBlock);
 	appendRawBlock(element, explorerBlock);
 }
+
+//Changes the document title according to the network the page is running on
```

Comment:
we said to turn it onto a function (e.g. `function buildPageTitle{}`) and call that immediately below it as `buildPageTitle();`.